server/views/pages/prisonerSelect.njk

```diff
@@ -36,7 +36,7 @@
       <a href="{{ pshUrl }}/prisoner/{{ prisoner.prisonerNumber }}" class="govuk-link">{{ prisoner.displayName }}</a>
     {% endset -%}
     {% set moveHospitalLinkHtml %}
-      <a href="/move-to-hospital/{{ prisoner.prisonerNumber }}" class="govuk-link" data-test="prisoner-move-to-hospital-link"><span class="govuk-visually-hidden">{{ prisoner.displayName }} - </span>Move to a hospital</a>
+      <a href="/move-to-hospital/{{ prisoner.prisonerNumber }}?returnUrl=/select-prisoner?searchTerm={{ searchTerm }}" class="govuk-link" data-test="prisoner-move-to-hospital-link"><span class="govuk-visually-hidden">{{ prisoner.displayName }} - </span>Move to a hospital</a>
```

Comment:
I would construct this link in the js: IMO putting the returnUrl in the njk makes things more confusing, given that the returnUrl processing code is in JS.